x-pack/plugins/lens/public/datatable_visualization/visualization.tsx

```diff
@@ -47,6 +47,9 @@ export const datatableVisualization: Visualization<DatatableVisualizationState>
       label: i18n.translate('xpack.lens.datatable.label', {
         defaultMessage: 'Data table',
```

Comment:
To be consistent with naming and match the designs.

```suggestion
        defaultMessage: 'Table',
```